app/src/userpages/components/StreamPage/Show/PreviewView/index.jsx

```diff
@@ -1,220 +1,90 @@
-        }))
+        isRunning: true,
+        isInspecting: false,
     }
 
-    subscribe = (stream: Stream) => {
-        this.subscription = this.props.client.subscribe({
-            stream: stream.id,
-            resend_last: this.state.visibleDataLimit,
-        }, (data, metadata) => this.onData({
-            data,
-            metadata,
-        }))
-    }
-
-    subscription: any
-
-    openInfoScreen = (d: DataPoint) => {
-        this.setState({
-            infoScreenMessage: d,
-        })
-    }
-
-    closeInfoScreen = () => {
+    onToggleRun = () => {
         this.setState({
-            infoScreenMessage: null,
+            isRunning: !this.state.isRunning,
         })
     }
 
-    pause = () => {
-        this.props.client.pause()
+    onInspectOpen = () => {
         this.setState({
-            paused: true,
+            isInspecting: true,
         })
     }
 
-    unpause = () => {
-        this.props.client.connect()
+    onInspectClose = () => {
         this.setState({
-            paused: false,
+            isInspecting: false,
         })
     }
 
     render() {
-        const tz = (this.props.currentUser && this.props.currentUser.timezone) || moment.tz.guess()
-        return (
-            <div>
+        const { stream } = this.props
+        const { isRunning, isInspecting } = this.state
+
+        if (stream) {
+            return (
                 <Fragment>
-                    Realtime Data Preview
-                    <div className="panel-heading-controls">
-                        {this.state.paused ? (
-                            <Button
-                                size="sm"
-                                color="primary"
-                                onClick={this.unpause}
-                                title="Continue"
-                            >
-                                Play
-                            </Button>
-                        ) : (
-                            <Button
-                                onClick={this.pause}
-                                title="Pause"
-                            >
-                                Pause
-                            </Button>
-                        )}
+                    <div className={styles.buttonContainer}>
+                        <Button color="userpages" className={styles.toggleButton} onClick={this.onToggleRun}>
+                            {!isRunning ?
+                                <SvgIcon name="play" className={styles.icon} /> :
+                                <SvgIcon name="pause" className={styles.icon} />
+                            }
+                        </Button>
+                        <Button color="userpages" onClick={this.onInspectOpen}>
+                            Inspect
+                        </Button>
                     </div>
+                    <StreamLivePreview
+                        key={stream.id}
+                        streamId="7wa7APtlTq6EC5iTCBy6dw"
```

Comment:
Ah, this is actually the tram data stream from production 😄 Accidentally committed it because I used it for testing. You can see data flowing if you set `STREAMR_WS_URL=wss://www.streamr.com/api/v1/ws` in your .env.